plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/file/IcebergFileMetastoreCatalogModule.java

```diff
@@ -33,6 +35,7 @@ protected void setup(Binder binder)
         binder.bind(IcebergTableOperationsProvider.class).to(FileMetastoreTableOperationsProvider.class).in(Scopes.SINGLETON);
         binder.bind(TrinoCatalogFactory.class).to(TrinoHiveCatalogFactory.class).in(Scopes.SINGLETON);
         binder.bind(MetastoreValidator.class).asEagerSingleton();
+        binder.bind(Key.get(boolean.class, HideDeltaLakeTables.class)).toInstance(true);
```

Comment:
Why `true`?

it used to be `false` by default

(we should have redirects instead)

Reply:
it looks like i was wrong, great